rockPaperScissors.py

```diff
@@ -42,16 +51,21 @@ def game_logic(player_1_choice, player_2_choice):
     
     return winner
 
+    
 # Setting global variables and initializing the players
 RPS = ['rock', 'paper', 'scissors']
 max_games = 3
 player = rps_game(0, '', '')
 computer = rps_game(0, '', '')
 game_count = 0
 
+# Pulls in my stored game statistics
+with open('game_stats.json','r') as game_statistics:
```

Comment:
if you happen to follow my advice regarding removing `game_stats.json`, you'll have to put logic in here to check whether or not the file exists and create it if it doesn't (because anyone cloning your repo won't automatically have that file). That's actually a really good exercise, since this is a really, really, really common thing to have to do.

Reply:
I am a disgrace to the git community. I beg a thousand apologies

Reply:
Also, is that all? I assumed I had some drastic error other than my failure to use git correctly...